easyconfigs/a/ANSYS/ANSYS-2022R2.eb

```diff
@@ -0,0 +1,30 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'ANSYS'
+version = '2022R2'
+
+homepage = 'https://www.ansys.com'
+description = """ANSYS simulation software enables organizations to confidently predict
+ how their products will operate in the real world. We believe that every product is
+ a promise of something greater."""
+
+toolchain = {'name': 'system', 'version': 'system'}
+
+# create a single zip file from contents of the 3 install ISO images.
+# Note that some of the content is duplicated across the ISOs but this can be overwritten
+# as required. N.B. ensure the copy operation also copies permissions.
+sources = [SOURCE_ZIP]
+checksums = [
+    '0aacfa18188c3c0499b19949493f4985820ab1dc06fb6c06c69ddeb433f49a5c',  # ANSYS-2022R2.zip
+]
+
+dependencies = [
+    ('libpng', '1.2.59'),
+]
+
+# To run the workbench "runwb2" certain deps are required, otherwise you get an error saying:
+# "System.DllNotFoundException: Ans.QT.dll":
+osdependencies = ['libxslt']
+
+modextravars = {'KMP_AFFINITY': 'disabled'}
```

Comment:
>N.B. the default Intel MPI doesn't work so commands will need to be forced to use Open MPI instead.

Is there an env var for that? If so, we should set it here.

Reply:
Unfortunately not (that I could see) and each command has a different way of specifying its MPI so it will need documenting instead.